csharp/HOLMS.Platform/HOLMS.Platform/Support/Currency/DollarCents.cs

```diff
@@ -34,7 +34,7 @@ public struct DollarCents : IComparable<DollarCents> {
 
         public static DollarCents Zero => new DollarCents(false, 0, 0);
 
-        public int TotalCents => (int)((IsNegative ? -1 : 1) * (Dollars * 100 + Cents));
+        public int TotalCents => (int)Math.Abs(Dollars * 100 + Cents);
```

Comment:
This isn't correct. Dollars and cents are uints on this class, with a boolean "IsNegative" field, so the Math.Abs doesn't apply here.